.github/workflows/publish-sdk-script-dry-run.yml

```diff
@@ -0,0 +1,82 @@
+name: Publish SDK NPM packages
+on:
+  # DELETE `push:` to only allow for manual trigering
+  push:
```

Comment:
i dont understand this. we are running on every push? 


this should be run when release is made on github.

Reply:
The current workflow executes when there is a push to `dry-run-wf` branch. That has to be deleted before setting the PR for `ready` (the comment above and also the points in the [PR description](https://github.com/celo-org/celo-monorepo/pull/10408#issue-1780472811)).

The other trigger is manual, but it will only be executed if `aaronmgdr`, `dckesler`, `alvarof2` or `jcortejoso` run the workflow. Should that be changed to run when release is made on github?

Reply:
we can remove dckesler as today is his last day. 

manual is ok for alpha or beta release but i think its best to only allow production releases to be done via github

Reply:
Changed!

For the releases, I added conditions for the release name to contain `SDK` (we don't want to trigger on any release) and extract the version from the tag, assuming tags are e.g. `sdks-4.1.0`. Would that be correct?

Reply:
yes that sounds great